css/dashboard.css

```diff
@@ -50,8 +59,26 @@
   opacity: 1;
 }
 
+.wallpaper-swatches {
+  flex-flow: row nowrap;
```

Comment:
`flex-flow` just combines `flex-direction` and `flex-wrap` https://css-tricks.com/snippets/css/a-guide-to-flexbox/